WARNING  root:server.py:36 Received SendMessage without client_id